src/ImageSharp/Processing/Transforms/Resamplers/Lanczos3Resampler.cs

```diff
@@ -14,6 +16,7 @@ public class Lanczos3Resampler : IResampler
         public float Radius => 3;
 
         /// <inheritdoc/>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Unfortunately, it doesn't help here. These method's are resolved dynamically through the interface, [so they can not be inlined](http://www.cs.technion.ac.il/users/yechiel/c++-faq/inline-virtuals.html).

Reply:
Dammit! I knew this, just forgot. wish we had a way around that.